### PR TITLE
Bug/fragility

### PR DESCRIPTION
**Pull request by rnugent3**
solution as discussed - except I also took the stages from the fragility function to make sure that we keep all of the information in both functions

### CODE REVIEW

File: HEC.FDA.ModelTest/unittests/PairedDataShould.cs
Comment (Brennan1994) [inline HEC.FDA.ModelTest/unittests/PairedDataShould.cs:95]: I'd like to see 3 test cases: one where the multiplier has fewer points than the base, one where it has more, and one where the coordinate count matches.

**Review by Brennan1994 — CHANGES_REQUESTED** [review]

File: HEC.FDA.Model/paireddata/PairedData.cs
Comment (Brennan1994) [inline HEC.FDA.Model/paireddata/PairedData.cs:238]: This is probably fine, but arrays are going to be more efficient than lists. If we find this to be slow when we do profiling, we may have to come back and refactor. List seem elegant to me for this solution so I'd leave em for now.

**Review by Brennan1994 — CHANGES_REQUESTED** [review]

File: HEC.FDA.ModelTest/unittests/PairedDataShould.cs
Comment (rnugent3) [reply HEC.FDA.ModelTest/unittests/PairedDataShould.cs:95]: great idea

**Comment by Brennan1994** [comment]
@rnugent3 waiting on that unit test update before I approve. Just bumping this to make sure you know I'm tracking this as in your court.

**Comment by rnugent3** [comment]
Thanks for the ping, Brennan.

**Comment by rnugent3** [comment]
@Brennan1994 this is ready for backcheck.

**Review by Brennan1994 — APPROVED** [review]

**Pull request merged**
